index.ts

```diff
@@ -232,7 +228,7 @@ try {
       filteredSnapshots
         .map((pkg) => `${globalInstallMessage} ${pkg.fullString}`)
         .join('\n') +
-      '\n```\n\n';
+      '\n```';
```

Comment:
These new lines were needed for both the global install message or package.json install message. So I moved them to line 248 if there is a `customMessageSuffix`.